### PR TITLE
Fix an isolated tag error

### PR DESCRIPTION
**Pull request by AngledLuffa**

### CODE REVIEW

File: not-to-release/sources/newsgroup/groups.google.com_herpesnation_c74170a0fcfdc880_ENG_20051125_075200.xml.conllu [hunk 1]
```diff
@@ -304,7 +304,7 @@
 3	healthy	healthy	ADJ	JJ	Degree=Pos	4	nsubj	4:nsubj	_
 4	treat	treat	VERB	VBP	Mood=Ind|Tense=Pres|VerbForm=Fin	12	advcl	12:advcl:when	_
 5	the	the	DET	DT	Definite=Def|PronType=Art	6	det	6:det	_
-6	sick	sick	NOUN	NN	Number=Sing	4	obj	4:obj	_
+6	sick	sick	ADJ	JJ	Number=Sing	4	obj	4:obj	_
```
Comment (nschneid) [inline not-to-release/sources/newsgroup/groups.google.com_herpesnation_c74170a0fcfdc880_ENG_20051125_075200.xml.conllu:307]: feature should be `Degree=Pos`

**Comment by AngledLuffa** [comment]
Derp, thanks

**Comment by AngledLuffa** [comment]
(fixed)

**Comment by nschneid** [comment]
Thanks! Will merge once #185 is done.

**Comment by AngledLuffa** [comment]
Can this wrapped up?  I think it probably won't cause too many conflicts even if #185 starts getting big

**Comment by sitalk** [comment]
Ц цена на  жж  г бз 6ф ф на нюц    ,, т3

**Pull request merged**